source/_posts/2025-07-18-why-i-disallow-facade-usage.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: post
+title: Why I disallow Laravel Facade usages
+category: Security
+tags: [ "PHP", "Laravel", "Facade", "Anti-Patterns" ]
+year: 2025
+month: 07
+day: 18
+published: true
+summary: "Laravel Facades: why I disallow their usage in my software project"
```

Comment:
"Disallow" is strong (maybe true, but...) - consider "Why I Avoid Laravel Facades" or "The Hidden Costs of Laravel Facades" for broader appeal